src/KeyringController.test.ts

```diff
@@ -1,17 +1,19 @@
-const HdKeyring = require('@metamask/eth-hd-keyring');
-const { normalize: normalizeAddress } = require('@metamask/eth-sig-util');
-const { strict: assert } = require('assert');
-const Wallet = require('ethereumjs-wallet').default;
-const sinon = require('sinon');
-
-const { KeyringController, keyringBuilderFactory } = require('..');
-const {
+import HdKeyring from '@metamask/eth-hd-keyring';
+import { normalize as normalizeAddress } from '@metamask/eth-sig-util';
+import type { Hex, Json, Keyring, KeyringClass } from '@metamask/utils';
+import { strict as assert } from 'assert';
+import Wallet from 'ethereumjs-wallet';
+import { restore, spy, stub, assert as sinonAssert } from 'sinon';
```

Comment:
It might be nice to import these as a group here for clarity. We could probably guess that `spy` and `stub` are coming from Sinon, but `restore` is rather ambiguous, and `assert` is a built-in module (you've renamed it of course, but maybe that's a sign). What do you think about:

``` typescript
import * as sinon from 'sinon';
```

Reply:
Thats a good point [a12ce45](https://github.com/MetaMask/KeyringController/pull/202/commits/a12ce456af2831238844dcc5cab8cda903378b2c)